docs/content/doc/packages/maven.en-us.md

```diff
@@ -81,6 +81,16 @@ To publish a package simply run:
 mvn deploy
 ```
 
+If you want to publish a prebuild package to the registry, you can use [`mvn deploy:deploy-file`](https://maven.apache.org/plugins/maven-deploy-plugin/deploy-file-mojo.html):
+
+```shell
+mvn deploy:deploy-file -Durl=https://gitea.example.com/api/packages/{owner}/maven -DrepositoryId=gitea -Dfile=/path/to/package.jar
```

Comment:
```suggestion
mvn deploy:deploy-file -Durl=$GITEA_URL/api/packages/$OWNER/maven -DrepositoryId=gitea -Dfile=$PATH_TO_PACKAGE.jar
```

Reply:
If the parameters get changed here we need to change them in all package docs.

I have no real preference how it should look like.

Reply:
Me neither, I simply thought that would be more fitting given that you use it in a fenced `shell` block, and hence the variables would be highlighted correctly.
However, I'd still add the URL and the PACKAGE_LOCATION as variables, because you have to look at the command right now to see that those need to be replaced as well.

Reply:
Not all code blocks are shell blocks but I may add another PR which changes the URLs for a placeholder.

Reply:
ps: there are a lot of "example.com" in docs, so maybe there should be some guideline about how to write documents with example URLs.